sample-kotlin/src/main/java/com/fblitho/lithoktsample/NavigatableDemoActivity.kt

```diff
@@ -30,29 +31,40 @@ open class NavigatableDemoActivity : AppCompatActivity() {
     super.onCreate(savedInstanceState)
 
     intent
-        .getIntExtra(INDEX, DEFAULT_INDEX)
-        .takeIf { it != DEFAULT_INDEX }
+        .getIntArrayExtra(INDICES)
+        .takeIf { it != null }
```

Comment:
`.filterNotNull` would allow not null `Int`s to go through. In this context, we need `takeIf` because we only need to check the array if it is not null in its entirety!

Reply:
Ah, gotcha! Thanks for clarifying.